components/Overlay/stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { Sample } from "../../lib/stories-common";
+
+import Overlay from "./index";
+
```

Comment:
Could add an `import "./stories.css"` here for story-specific CSS (i.e. the example popup)